common/src/test/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthControllerTest.kt

```diff
@@ -130,6 +134,20 @@ class NativeAuthControllerTest {
     @get:Rule
     var rule: MockitoRule = MockitoJUnit.rule()
 
+    companion object {
+        @BeforeClass
+        @JvmStatic
+        fun setupClass() {
+            BuildValues.setUseRealAuthority(true)
```

Comment:
Will use new method signature for `setUseRealAuthority` when its refactoring PR gets merged.